mz_bokeh_package/components/plot_settings.py

```diff
@@ -704,6 +713,24 @@ def _init_plot_settings_values(self):
             self._set_setting_property(setting_id, value)
             self._set_setting_widget_value(setting_id, value)
 
+    def _set_color_for_renderer(self, renderer_name, new_color):
```

Comment:
As mentioned above: Try if you can change that to 
```suggestion
    def _set_color_for_renderer(self, renderer: bokeh.models.renderes.GlyphRenderer, new_color: ...):
```

Reply:
While typing this, PyCharm detected bokeh but when running it gives an error:
`name 'bokeh' is not defined`